packages/global.d.ts

```diff
@@ -9,3 +9,7 @@ declare var __VERSION__: string
 // Feature flags
 declare var __FEATURE_OPTIONS__: boolean
 declare var __FEATURE_SUSPENSE__: boolean
+
+// Global opration
+declare type AnyFunction = (...args: any[]) => any
```

Comment:
I don't like this. Magic global convenience types can make it really confusing for others reading the code.